app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_ls = command.split()
+    file_old, file_new = command_ls[1], command_ls[2]
```

Comment:
u can write it like that command_cp, file_old, file_new = command.split() in one line)